test_scripts/Resumption/Handling_errors_from_HMI/015_resume_one_rpc_without_response_unexpected_disconnect.lua

```diff
@@ -0,0 +1,56 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+--
+-- Description:
```

Comment:
Core might need to revert the resumed data in case of no response as well as a late response